water/modules/load_data.py

```diff
@@ -82,26 +82,27 @@ def load_pollution_dynamics():
     # Determine max and min pollution values and all scenario names
     max_pols = []
     min_pols = []
-    scenarios = []
+    pollution_nodes = []
     for key, df in pollution.items():
         if key != 'chemical_start_time':
-            scenarios.append(key)
+            pollution_nodes.append(key)
             v = df.values.ravel()
             max_pols.append(np.max(v))
             min_pols.append(np.min(v[v > 0]))
     max_pol = np.max(max_pols)
     min_pol = np.min(min_pols)
 
     # Choose a default node for pollution injection
-    start_node = scenarios[0]
+    start_node = pollution_nodes[0]
 
     # Determine the step numbers for the beginning and end of the pollution
-    # data. This assumes all pollution scenarios are identical in time to the
-    # default starting node!
+    # data. This assumes all pollution pollution_nodes are identical in time
+    # to the default starting node!
```

Comment:
When we decide on the name, this comment should be updated.

'This assumes all pollution pollution_nodes are identical...'